modules/pel/peltool/README.md

```diff
@@ -0,0 +1,67 @@
+### Peltool usage
+To know more about peltool capabilities using the help command as shown below
+`Peltool.py --help`
+
+#### List PELs
+- Get list of PELs : `peltool.py -l` (By default serviceable PELs are listed)
+   **Note:** For non-BMC environment user must pass the PEL directory e.g. `peltool.py -p <pel_dir> -l`
+- Get list of PELs including serviceable PELs : `peltool.py -ls`
+- Get list of PELs including non-serviceable PELs: `peltool.py -lN`
+- Get list of PELs including hidden PELs: `peltool.py -lH`
+- Get list of PELs including terminating PELs: `peltool.py -lt`
+
+##### List PELs based on severity and only options
+
+- Get PEls including PELs with severity as Informational: `peltool.py -l -S Informational`
+- **Note:** User can filter the PELs by severity value from: `{Informational, Recovered, Predictive, Unrecoverable, Critical, Diagnostic, Symptom}`. Can provide multiple values e.g. `peltool.py -l -S Informational Critical`
```

Comment:
I would just mention see usage `peltool.py -h` to get supported severities to filter so that we no need to worry about update this if a new severity added in the future. 

> Can provide multiple values e.g. `peltool.py -l -S Informational Critical`

We can provide multiple severities e.g. `peltool.py -l -S Informational Critical`